src/shared/bus-unit-util.c

```diff
@@ -1308,6 +1308,23 @@ static int bus_append_resource_limit(sd_bus_message *m, const char *field, const
         return 1;
 }
 
+static int bus_append_smack_stuff(sd_bus_message *m, const char *field, const char *eq) {
```

Comment:
"this smack stuff" is also aa stuff...

maybe call it "mac_stuff"? or "lsm_stuff"?